Tiltfile

```diff
@@ -22,7 +22,7 @@ settings = {
     "capi_version": "v1.5.3",
     "cert_manager_version": "v1.13.1",
     "kubernetes_version": "v1.28.0",
-    "aks_kubernetes_version": "v1.26.3",
+    "aks_kubernetes_version": "v1.27.3",
```

Comment:
Q: is this version bump what enables the support for marketplace extensions?

Reply:
Marketplace extensions are enabled by ASO 2.4.0. I had to bump the k8s version because I can't create AKS clusters in Tilt on 1.26.3. I think this might be due to the new ASO version but I'm not entirely sure.